package.json

```diff
@@ -44,9 +44,8 @@
     "pretest": "npm run compile",
     "posttest": "npm run check",
     "docs-test": "linkinator docs -r --skip www.googleapis.com",
-    "predocs-test": "npm run docs",
-    "generate-scaffolding": "repo-tools generate all && repo-tools generate lib_samples_readme -l samples/ --config ../.cloud-repo-tools.json"
-  },
+    "predocs-test": "npm run docs"
```

Comment:
you can delete ` "@google-cloud/nodejs-repo-tools": "^3.0.0",` from below as well now!

Reply:
Gotcha.